### PR TITLE
Pull request for my Own review

### PR DESCRIPTION
**Pull request by desorgui**
This project was created during my trial at the Microverse admission.

### CODE REVIEW

**Review by desorgui — COMMENTED** [review]
### Required

- I think your readme file should describe your project.
- I think that you must use the correct Github flow.

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @desorgui , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

Hi great job with your review. 
-  [ ] Please as part of the requirements, you are supposed to provide at least 2 inline comments. You can do that by going to "files changed", and selecting the part of the code that can be improved 


https://user-images.githubusercontent.com/37901469/166647112-e44632ed-b32e-46be-b198-44dac86a7638.mov


https://user-images.githubusercontent.com/37901469/166647133-10c2d8a3-ebdc-48d7-9fe9-1e212a182519.mov

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.responsive {
+	width: auto;
```
Comment (desorgui) [inline css/style.css:2]: - [ ] Kindly make sure install [Linter](https://github.com/microverseinc/linters-config/tree/master/html-css) to better address indentation issues.

File: babel/index.babel [hunk 2]
```diff
@@ -0,0 +1,27 @@
+// !! IMPORTANT README:
+
```
Comment (desorgui) [inline babel/index.babel:2]: Kindly try to change these contents in your readme file.

**Review by desorgui — COMMENTED** [review]
Good job so far!
There are some issues that you still need to work on but you are almost there!

**Review by ShahierNashaat — APPROVED** [review]
Hi @desorgui , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**